operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -538,6 +541,17 @@ private void buildIngressController(String name, String domain,
                 .endSpec();
         }
 
+        // if configured for profiles, move the ingress replicas to the default machine pool by adding a dummy toleration
+        if (OperandUtils.shouldProfileLabelsExist(informerManager.getLocalAgent())
+                && overrideManager.migratedToDynamicScalingScheduling()) {
+            builder
+                .editSpec()
+                    .editOrNewNodePlacement()
+                        .addNewToleration("NoSchedule", "kas-fleetshard-ingress", "Exists", null, null)
+                    .endNodePlacement()
+                .endSpec();
+        }
```

Comment:
I don't understand this part - wouldn't the `NoSchedule` taint on the profile-specific nodes push them off? Is this just to trigger _some_ change to the IngressController CR to re-evaluate the taints?

Reply:
> wouldn't the NoSchedule taint on the profile-specific nodes push them off?

No, NoSchedule will just prevent new placements.  The existing will remain in place.

> Is this just to trigger some change to the IngressController CR to re-evaluate the taints?

To force the ingress replica pods to reschedule yes, which will force them to the default pool.